cloudify_aws/common/utils.py

```diff
@@ -574,3 +581,175 @@ def raise_on_substring(iface,
         if any(substring in message for substring in substrings):
             raise raisable(message)
         return {}
+
+
+def with_rest_client(func):
```

Comment:
This is the same implementation for this decorator as in starlingX plugin. I recommend moving this code to utilities_sdk as a common function. It will be easier to maintain this code.